docs/widgets/extra/keyboard.md

```diff
@@ -32,6 +32,10 @@ To set the mode manually, use `lv_keyboard_set_mode(kb, mode)`. The default mode
 You can assign a [Text area](/widgets/core/textarea) to the Keyboard to automatically put the clicked characters there.
 To assign the text area, use `lv_keyboard_set_textarea(kb, ta)`.
 
+### Key Popovers
+To enable key popovers on press, like on common Android and iOS keyboards, use `lv_keyboard_set_popovers(kb, true)`. The default control maps are preconfigured to only show the popovers on keys that produce a symbol and not on e.g. space. If you use a custom keymap, set the `LV_BTNMATRIX_CTRL_POPOVER` flag for all keys that you want to show a popover.
+
```

Comment:
(I would just add this myself but I don't want to disrupt your force-push.)

I think we should add a note that the selectable extra characters (e.g. accents) are currently not implemented, and this is just a visual effect, to preempt the question being asked on the forum.

Reply:
Good idea. I added a corresponding phrase.

Question for the future: are force-pushes ok or would you rather have me add further commits for follow-up changes?